src/main/java/core/basesyntax/strategy/PurchaseStrategy.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+
+public class PurchaseStrategy implements OperationStrategy {
+    @Override
+    public void apply(String fruit, int quantity, Storage storage) {
```

Comment:
Receive Storage here in method not good practice, better when you invoke from Storage class

Reply:
refactored